seabird_chat_ingest.proto

```diff
@@ -0,0 +1,144 @@
+syntax = "proto3";
+
+option go_package = ".;pb";
+
+import "common.proto";
+
+package seabird;
+
+// Note that all of the types in this file end with ChatRequest or end with
+// ChatEvent to prevent namespace conflicts between the main seabird.proto file.
+
+// SendMessageChatRequest requests for a message to be sent to a given channel.
+// Either SuccessChatEvent or FailedChatEvent SHOULD be returned in response if
+// possible. This MUST NOT cause any chat events from the bot to appear in the
+// output event stream.
```

Comment:
I don't understand the third sentence of this comment:
```
// ...This MUST NOT cause any chat events from the bot to appear in the
// output event stream.
```

What does that mean?